src/components/InputField/InputField.js

```diff
@@ -76,8 +76,9 @@ class InputField extends Component {
   }
   render() {
     const errorClassName = this.props.showErrorMessage === true ? 'form__field-error-wrapper' : '';
+    const showBackgroundClassName = this.props.setBackgroundColor === true && this.props.type === 'checkbox' ? 'form__field-wrapper--background' : '';
     return (
-      <div id={`field-wrapper--${this.props.id}`} className={`form__fieldset form__field--wrapper form__field-wrapper--${this.props.type} ${errorClassName} ${this.props.extraClass ? this.props.extraClass : ''} `}>
+      <div id={`field-wrapper--${this.props.id}`} className={`form__fieldset form__field--wrapper form__field-wrapper--${this.props.type} ${errorClassName} ${showBackgroundClassName} ${this.props.extraClass ? this.props.extraClass : ''} `}>
```

Comment:
should we move the extraClass check out of this line and use a variable like you did with the errorClassName and showBackgroundClassName too?

Reply:
We can do